torch/fx/graph_module.py

```diff
@@ -56,12 +70,20 @@ def __init__(self, body):
             super().__init__()
             self.__dict__ = body
 
-    try:
-        CodeOnlyModule.forward = _forward_from_src(body['_code'])
-    except KeyError:
-        # BC: attribute name was changed from `code` to `_code` to facilitate
-        # making `code` into a property and adding a docstring to it
-        CodeOnlyModule.forward = _forward_from_src(body['code'])
+    if importer is not None:
+        # If we are unpickling this module from a torch.package, we can
+        # retrieve the serialized forward directly from the package.
+        module_id = body['_fx_generated_id']
+        CodeOnlyModule.forward = importer.import_module(f"fx-generated._{module_id}").forward
+    else:
+        # Otherwise, we're unpickling from a regular pickle file. Retrieve the
+        # serialized forward that is stashed in the object state.
+        try:
+            CodeOnlyModule.forward = _forward_from_src(body['_code'], importer)
```

Comment:
I know it was here before, but copypasta with unnecessary exception: `body.get('_code') or body['code']`